src/test/LibsodiumEncryptor.test.ts

```diff
@@ -24,7 +24,7 @@ describe("LibsodiumEncryptor", () => {
     const dataKeyProvider = await FixedDataKeyProvider.forLibsodium()
     const { dataKey } = await dataKeyProvider.generateDataKey()
 
-    const { encryptedItem, nonce } = await encryptor.encrypt({
```

Comment:
This is a small improvement that's unrelated to the other PR, I just noticed we were awaiting for no reason on some things.